crates/ironrdp-client/src/rdp.rs

```diff
@@ -131,10 +177,21 @@ async fn connect(
 
     let mut framed = ironrdp_tokio::TokioFramed::new(socket);
 
+    let mut drdynvc =
+        ironrdp::dvc::DrdynvcClient::new().with_dynamic_channel(DisplayControlClient::new(|_| Ok(Vec::new())));
+
+    // Instantiate all DVC proxies
+    for proxy in config.dvc_pipe_proxies.iter() {
+        let channel_name = proxy.channel_name.clone();
+        let pipe_name = proxy.pipe_name.clone();
+
+        trace!(%channel_name, %pipe_name, "Creating DVC proxy");
+
+        drdynvc = drdynvc.with_dynamic_channel(dvc_pipe_proxy_factory.create(channel_name, pipe_name));
+    }
+
```

Comment:
question: Is it best to instantiate the DVC proxies after opening the TCP stream to the server, or it does not matter? If there is a rationale, please write it down as an inline comment please

Reply:
I don't think this matters much; Historically we are already performing other DVC/SVC initialization after the socket connection, I just have added another dvc to the list